src/sagas/repository.js

```diff
@@ -96,7 +96,11 @@ function* pollJobWorker(jobId, jobTypeSuccess, jobTypeFailure) {
       yield call(pollJobWorker, jobId, jobTypeSuccess, jobTypeFailure);
     }
   } catch (err) {
-    showNotification(err);
+    if (err.response) {
```

Comment:
This logic gets repeated a bunch...would it make sense to have it in the `showNotification` function?